crypto3/libs/algebra/test/fields_static.cpp

```diff
@@ -134,10 +110,6 @@ BOOST_AUTO_TEST_CASE(field_operation_test_bls12_381_fr) {
                                       0x53530b5d9374f4f45c1b2267998b8cd7d1bf6d19a90564fb898ed8970544ea28_cppui_modular255};
     constexpr const_set_t constants1 = {811706348};
 
-    // TODO: the reason of the error "function parameter 'elements' with unknown value cannot be used in a constant
```

Comment:
It's a bit strange this did not compile. Can you please re-try? Maybe we need to remove the const reference for elements1?

Reply:
I have found a trick that worked: we need to have values `static` and pass them as template parameters by value to make that function work.
This introduced some caveat, I can't figure out how to properly check presence of `.sqrt()` method.